homework/index.js

```diff
@@ -23,28 +23,73 @@
       if (key === 'text') {
         elem.textContent = value;
       } else {
-        elem.setAttribute(key, value);
+        elem.setAttribute(key, value); // ?
       }
     });
     return elem;
   }
 
+  function changeDateTimeFormat(dateTime) {
+    const timeFormat = new Date(dateTime);
+    return timeFormat.toLocaleString();
+  }
+
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const list = createAndAppend('li', ul, { class: 'topClass' });
+    const name = createAndAppend('li', list, {
+      text: 'Repository :',
+      class: 'keys',
+    });
+    const description = createAndAppend('li', list, {
+      text: 'Description :',
+      class: 'keys',
+    });
+    const fork = createAndAppend('li', list, {
+      text: 'Forks :',
+      class: 'keys',
+    });
+    const time = createAndAppend('li', list, {
+      text: 'Updated :',
+      class: 'keys',
+    });
+
+    createAndAppend('a', name, {
+      text: repo.name,
+      href: repo.html_url,
+      class: 'values',
+    });
+    createAndAppend('a', description, {
```

Comment:
You should only use `<a>` tags if you have a corresponding URL in an `href` attribute. Without it, it doesn't link to anything.